src/components/MultiHopTrade/hooks/useGetTradeQuotes/useGetTradeQuotes.tsx

```diff
@@ -147,7 +152,51 @@ export const useGetTradeQuotes = () => {
     [isSnapshotApiQueriesPending, votingPower],
   )
 
+  const isSnapInstalled = useIsSnapInstalled()
+  const walletSupportsBuyAssetChain = useWalletSupportsChain({
+    chainId: buyAsset.chainId,
+    wallet,
+    isSnapInstalled,
+  })
+  const isBuyAssetChainSupported = walletSupportsBuyAssetChain
+
+  const shouldRefetchTradeQuotes = useMemo(
+    () =>
+      Boolean(
+        wallet &&
+          sellAccountId &&
+          sellAccountMetadata &&
+          isBuyAssetChainSupported &&
+          receiveAddress &&
```

Comment:
We shouldn't need this anymore, checking for `receiveAddress` will be enough, and this would rug manual receive address

Reply:
eff62f5e7ea6aad4082bbfaa1b57ec3f6fb88bee